.github/scripts/build-gcc.sh

```diff
@@ -18,14 +18,17 @@ if [ $RUN_CONFIG = 1 ] || [ ! -f "$GCC_BUILD_PATH/Makefile" ] ; then
                 --enable-threads=win32 \
                 --disable-win32-registry"
             ;;
+        *linux*)
+            HOST_OPTION=" \
+                --includedir=$TOOLCHAIN_PATH/$TARGET/include"
```

Comment:
I would keep it as it was before for wider scope until it is really needed to be excluded